src/xdocs/config_imports.xml

```diff
@@ -2546,6 +2546,52 @@ class FooBar {
         <source>
 &lt;module name=&quot;UnusedImports&quot;/&gt;
         </source>
+        <p>Example:</p>
+        <source>
+package com.example;
+
+import com.example.Demo;  // violation, class from the same package should not be imported
+
+import java.util.Scanner; // OK
+import java.util.Scanner; // violation, class should not be imported more than once
+
+import java.lang.String; // violation, imported from "java.lang" package
+
+import java.util.Stack;  // OK
+import java.util.Map;   // violation, this class is not referenced
+
+Stack stack = new Stack();
```

Comment:
Please make code same as you used in CLI. It must be compile able

Reply:
@TanayMorakhia , please do not resolve items of reviewer, just reply "done" for each review item